examples/hands_on_opencl/ex08/matmul.jl

```diff
@@ -159,7 +159,7 @@ for i in 1:COUNT
         global_size = (Ndim,)
         local_size = (div(ORDER, 16),)
 
-        evt = clcall(mmul, Tuple{Int32, Int32, Int32, Ptr{Float32}, Ptr{Float32}, Ptr{Float32}, cl.LocalMem{Float32}},
+        evt = clcall(mmul, Tuple{Int32, Int32, Int32, Ptr{Float32}, Ptr{Float32}, Ptr{Float32}, CLPtr{Float32}},
```

Comment:
All these `Ptr`s have to be `CLPtr`, the fact that this works now is a bug...

Reply:
https://github.com/JuliaGPU/OpenCL.jl/issues/294

Reply:
I changed all the `Ptr` to `CLPtr` for the `CLArray` and checked locally that the 3 examples still work

Reply:
Great, thanks!